File: Easy/SquareOfSortedArray.py
# https://leetcode.com/problems/squares-of-a-sorted-array/

class Solution(object):
    def sortedSquares(self, nums):
        i=0
        while(i<len(nums)):
            nums[i] = nums[i] ** 2
            i+=1
        nums.sort()
        return nums
        

File: Easy/SquareOfSortedArray.java
package Easy;
// https://leetcode.com/problems/squares-of-a-sorted-array/
import java.util.Arrays;

public class SquareOfSortedArray {
    public static int[] sortedSquare(int[] arr){
        int[] result = new int[arr.length];
        int i=0;
        while(i++<=arr.length) {
            result[i] = arr[i]*arr[i]; 
        }
        Arrays.sort(result);
        return result;
    }
}
